testing/environments/docker/logstash/setup.sh

```diff
@@ -1,18 +1,9 @@
 #!/usr/bin/env bash
 
-# setup logstash dependencies
-set -x && \
-    apt-get update && \
-    apt-get install -y --no-install-recommends logrotate && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
 # download & install logstash
```

Comment:
In case that are all the lines left, we probably can remove  the setup.sh script and move everything back to the docker file.